### PR TITLE
GH-2379 Comms use fetch instead of request

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-2379-Fetch

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review

**Comment by GordonSmith** [comment]
@jbrundage bump

File: packages/common/src/Class.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class Class {
     }
 
     cssTag(id: string): string {
-        return id.replace(/[^a-z0-9]/g, (s) => {
+        return ("" + id).replace(/[^a-z0-9]/g, (s) => {
```
Comment (jbrundage) [inline packages/common/src/Class.ts]: Why was this necessary?

Comment (GordonSmith) [reply packages/common/src/Class.ts]: I think its for the case where id is a number.
